common/app/model/content/Atoms.scala

```diff
@@ -206,9 +207,13 @@ final case class TimelineItem(
 
 object Atoms extends common.Logging {
 
-  def extract[T](atoms: Option[Seq[AtomApiAtom]], extractFn: AtomApiAtom => T): Seq[T] = {
+  def extract[T](
+    atoms: Option[Seq[AtomApiAtom]], 
+    extractFn: AtomApiAtom => T, 
+    filterFn: T => Boolean = { _: T => true }
```

Comment:
why injecting the filter as an extract argument rather than letter the caller filter the resulting `Seq`? Also is it used somewhere?

Reply:
Fair point! I was caught in the heat of the moment and did not see that. Thanks!